src/main/java/org/cloudfoundry/promregator/scanner/Instance.java

```diff
@@ -8,12 +8,21 @@
 public class Instance {
 	private ResolvedTarget target;
 	private String instanceId;
-	private String accessUrl;
+  private String accessUrl;
+  private boolean isInternal;
```

Comment:
beans of type boolean do not have the prefix `is`.
```suggestion
  private boolean internal;
```

Reply:
fixed